styles/styles.css

```diff
@@ -555,4 +555,149 @@ header .wrapper .cocktailForm h1 {
           transform: translate(3px, 3px);
   -webkit-box-shadow: 1px 1px 0px #e4ae5d;
           box-shadow: 1px 1px 0px #e4ae5d;
+}
+
+@media (max-width: 1020px) {
+  header .wrapper {
+    position: relative;
+  }
+  header .wrapper .imageContainer {
+    position: absolute;
+    width: 360px;
+    left: -20px;
+    bottom: -20px;
+    z-index: -10;
+  }
+  header .wrapper .cocktailForm {
+    width: 100%;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+        -ms-flex-direction: column;
+            flex-direction: column;
+    -webkit-box-align: end;
+        -ms-flex-align: end;
+            align-items: flex-end;
```

Comment:
What does this stuff do? I'm not familiar with these properties

Reply:
They're just cross-browser compatibility - I have an extension that automatically in-fills anything needed to make it work on ie and other browsers.